acceptance/tests/metrics/metrics_test.go

```diff
@@ -21,16 +22,19 @@ const StaticClientName = "static-client"
 // Test that prometheus metrics, when enabled, are accessible from the
 // endpoints that have been exposed on the server, client and gateways.
 func TestComponentMetrics(t *testing.T) {
-	t.Skipf("Skipping this test because it's not yet supported with agentless")
 	env := suite.Environment()
 	cfg := suite.Config()
 	ctx := env.DefaultContext(t)
 	ns := ctx.KubectlOptions(t).Namespace
 
 	helmValues := map[string]string{
+		// Remove before merging
+		"global.imageConsulDataplane":       "curtbushko/consul-dataplane:latest",
+		"global.imageK8S":                   "curtbushko/consul-k8s-control-plane-dev:latest",
 		"global.datacenter":                 "dc1",
 		"global.metrics.enabled":            "true",
 		"global.metrics.enableAgentMetrics": "true",
+		"client.enabled":                    "true",
```

Comment:
Could you add a comment for why we're enabling client here?

Reply:
Done!